factory.py

```diff
@@ -317,13 +402,20 @@ def ssh(self, cmd=None):
 
         self.invoke_ssh(ssh_command)
 
+    def console(self):
+        self.invoke_console(self.var / 'vm.mon')
+
 
 def add_vm_arguments(parser):
+    parser.add_argument('-i', '--image', default=DEFAULTS[get_arch()])
     parser.add_argument('--share', action='append', default=[])
     parser.add_argument('-m', '--memory', default=512, type=int)
-    parser.add_argument('-p', '--smp', default=1, type=int)
+    parser.add_argument('-s', '--smp', default=1, type=int)
```

Comment:
Sure, change them all you want, they're not used or documented anywhere anyway. I'd even say we'd be better off removing all the short options here.